src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -234,6 +235,15 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
         }
     }
 
+    @Override
+    protected @Nonnull String getSuccessfulDestination() {
+        FolderComputation<I> computation = getComputation();
+        if (computation != null) {
```

Comment:
This is other subject. IMO, it should be addressed in core, _how Jenkins handles this kind of dynamic content using ajax_.

This is completely unrelated with this issue. We should not condition this redirecction to computation is started (scheduled). What happend if another implementation of `ComputedFolder` wants to override `getSuccessfulDestination` and redirects to Jenkins home? Does it make sense to wait for?